packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -86,6 +86,9 @@ import InMeetingActions from './in-meeting-actions';
  * @property {String} [resourceId]
  * @property {String} [pin]
  * @property {Boolean} [moderator]
+ * @property {String|Object} [meetingQuality]
```

Comment:
We decided to make meetingQuality either a string or a object so that the value can be set together or individually